src/main/java/org/apache/maven/doxia/DefaultConverter.java

```diff
@@ -357,23 +458,19 @@ private void parse( File inputFile, String inputEncoding, String inputFormat, Ou
         }
 
         File outputFile;
-        if ( output.getFile().isDirectory() )
+        if ( outputDirectoryOrFile.isDirectory() 
+             || !SelectorUtils.match( "**.*", output.getFile().getName() ) 
```

Comment:
This pattern looks wrong. Shouldn't it be: `**/*`?

Reply:
Rather `*.*` because it only matches on the file name (not the full path), however I haven't modified the globbing either. Let us not put too much into this PR.

Reply:
Agreed.